overlord/snapstate/export_test.go

```diff
@@ -74,3 +74,27 @@ func MockReadInfo(mock func(name string, si *snap.SideInfo) (*snap.Info, error))
 	readInfo = mock
 	return func() { readInfo = snap.ReadInfo }
 }
+
+func RunDoHandler(mgr *SnapManager, kind string, t *state.Task) error {
```

Comment:
Ah this is pretty interesting. Nice way to expose access to private handler functions.